demisto_sdk/commands/download/downloader.py

```diff
@@ -478,7 +478,11 @@ def merge_and_extract_existing_file(self, custom_content_object: dict) -> None:
             searched_basename: str = self.get_searched_basename(file_name, ex_file_ending, ex_file_detail)
             corresponding_pack_file_object: dict = self.get_corresponding_pack_file_object(searched_basename,
                                                                                            corresponding_pack_object)
-            corresponding_pack_file_path: str = corresponding_pack_file_object['path']
+            if not corresponding_pack_file_object:
+                corresponding_pack_file_path: str = os.path.join(self.output_pack_path, file_entity,
+                                                                 self.create_dir_name(file_name), searched_basename)
```

Comment:
Can you explain what is happening here?

Reply:
If the file doesn't exist, we combine all the path parts and download the file to the path we've just created.

Reply:
Add an example comment on what is each path part. Will make the future debugging easier.
if an image doesn't exist how can you create one?

Reply:
I download the unified intergation, extract it and then I have the image, I just have to create the path to download the image to.